service/policy/db/queries/obligations.sql

```diff
@@ -584,3 +584,66 @@ WHERE obligation_value_id = $1;
 DELETE FROM obligation_triggers
 WHERE id = $1
 RETURNING id;
+
+-- name: listObligationTriggers :many
+SELECT
+    JSON_STRIP_NULLS(
+        JSON_BUILD_OBJECT(
+            'id', ot.id,
+            'obligation_value', JSON_BUILD_OBJECT(
+                'id', ov.id,
+                'value', ov.value,
+                'obligation', JSON_BUILD_OBJECT(
+                    'id', od.id,
+                    'name', od.name,
+                    'namespace', JSON_BUILD_OBJECT(
+                        'id', n.id,
+                        'name', n.name,
+                        'fqn', COALESCE(ns_fqns.fqn, '')
```

Comment:
Do we need to COALESCE this? It shouldn't be null?

Reply:
Discussed: this is an established pattern we know is working, so we will not use a different approach until we reconsider fqn reindexing/cache consistency state.